### PR TITLE
[VIRTS-2876] Operation event-logs + enable/disable agent output

### PR DESCRIPTION
**Pull request by bleepbop**
## Description
Added endpoint for retrieving event-logs from an operation. For both the `GET /api/v2/operation/{op_id}/report` and `GET /api/v2/operation/{op_id}/event-logs` endpoints, users can now request that agent output be included in the report/event logs. This can be done by adding a request body with the following: `{'enable_agent_output': true/false}`. Unit tests have been updated accordingly.

## Type of change

Please delete options that are not relevant.
- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Tested manually with postman, and updated operations API unit tests.

## Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: app/api/v2/handlers/operation_api.py [hunk 1]
```diff
@@ -143,6 +157,14 @@ async def get_potential_links_by_paw(self, request: web.Request):
         potential_links = await self._api_manager.get_potential_links(operation_id, access, paw)
         return web.json_response(potential_links)
 
+    '''Helper Methods'''
```
[the remark this answers is not in the document]
Comment (bleepbop) [reply app/api/v2/handlers/operation_api.py]: Updated this to have the above changes!

**Comment by alexanderkent** [comment]
Nice to see the appropriate unit tests for this too 👍🏻   

get_event_logs

I think having a GET request submit a payload e.g. `{'enable_agent_output': true/false}` is anti-pattern and potentially non-compliant with the HTTP specs. 

From the HTTP 1.1 specs
https://datatracker.ietf.org/doc/html/rfc7231#page-24
`
A payload within a GET request message has no defined semantics; sending a payload body on a GET request might cause some existing implementations to reject the request.
`

Have we considered using a POST request for these instead of GET?

**Comment by bleepbop** [comment]
> Nice to see the appropriate unit tests for this too 👍🏻
> 
> get_event_logs
> 
> I think having a GET request submit a payload e.g. `{'enable_agent_output': true/false}` is anti-pattern and potentially non-compliant with the HTTP specs.
> 
> From the HTTP 1.1 specs
> https://datatracker.ietf.org/doc/html/rfc7231#page-24
> `A payload within a GET request message has no defined semantics; sending a payload body on a GET request might cause some existing implementations to reject the request.`
> 
> Have we considered using a POST request for these instead of GET?

That's a good point! Changed both endpoints to be POST requests for consistency with the rest of the API.

**Review by elegantmoose — APPROVED** [review]
Nice work

**Review by wbooth — APPROVED** [review]

**Pull request merged**